### DISCUSSION TITLE
[Projects Beta] Copy Project Board

### DISCUSSION THREAD

**Discussion by kelipropp** [community/community#8357] [Projects and Issues] [answered]
I was wondering if there is a way to copy the project board with all the same workflows as a previous beat board that has been set up.

**Comment by tn3rb** [comment]
wondering the same.
an alternate but equally acceptable option would be the ability to use an existing project and all of it's workflows/labels/settings/etc as a template that new projects can be created from.

**Comment by garthkoyle** [comment]
When cloning, or duplicating or copying a (beta) project, it would be nice to choose which items to maintain association with the new project. E.g. select the items (e.g. open or closed items)

**Comment by mlcaraldi** [comment]
The ability to copy a project would be really useful!

**Comment by MichaelTiemannOSC** [comment]
How about this: https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/copying-a-project-board

**Reply by serpro69** [reply]
This is not related to "Projects Beta"

**Reply by MichaelTiemannOSC** [reply]
Sorry!  Thought it was.  Nevermind...

**Reply by kelipropp** [reply]
Thanks though. I appreciate the help.

**Comment by mulle-nat** [comment]
Projects beta is really neat. But with the WiKi I have no problems entering important data into it, because I can clone it and keep a local backup. With the Projects beta though, there is no way to git clone it (which would be the best option to access the data). Now I would have to copy/paste ... a lot. I am somewhat betting on Projects beta getting a way to export my data, because otherwise I'd rather use a different tool in the long run.

**Reply by serpro69** [reply]
Agree all the way. I'm using projects beta and now and then can't wonder what if something happens to my data. So at least for a peace of mind I really need an ability to export all data and have a local copy of it. Having `git clone` option would be best, but I would be content with other solutions so long as I can backup my data and have it locally in a human-readable text format.

**Comment by tbradsha** [comment]
+1

**Comment by mitchelsellers** [comment]
I would love to be able to copy a project structure ONLY, I wouldn't want any issues to come across but all views, fields, lists, etc would be huge.

**Reply by kelipropp** [reply]
Yes this is what I would like too. I would like to also add that all the workflows should be also copied.

**Reply by dirtybike** [reply]
This is huge. All custom columns need to be recreated today.

**Comment by nk-rakanishu** [comment]
+1

**Comment by 700software** [comment]
It's so easy to delete a column by accident instead of hiding it. Need ability to create backup copy for that reason. (although undo delete column would be a possible solution, there's still other things that can go wrong)

**Comment by yshuman1** [comment]
+1 for ability to copy a project board

**Reply by lkaragol** [reply]
+1

**Comment by frouioui** [comment]
This would be really useful.

**Comment by fernandascovino** [comment]
up! this is a useful feature for organizing multiple teams and products, having the ability to create template boards

**Comment by ehicks71** [comment]
It would be nice if copying project also copied the workflow settings.

**Reply by wmalexander** [reply]
Agreed!

**Comment by andersan** [comment]
I think this was released a month ago: https://github.blog/changelog/2023-01-19-github-issues-january-19th-update/

Here's the [instructions](https://docs.github.com/en/issues/planning-and-tracking-with-projects/creating-projects/copying-an-existing-project). 

We're migrating a project from one github organization to another. Does anyone know if there is a way to get these project views to copy and have the issues in the same place in the boards? We're transferring the repo for the project as well, so those issues will go along with it.

**Comment by rileybroughten** [comment, marked as answer]
👋 As noted above, you can copy a project to include views, custom fields, and if you want, draft issues. The changelog announcement for that is found [here](https://github.blog/changelog/2023-01-19-github-issues-january-19th-update/#%F0%9F%96%A8%EF%B8%8F-get-started-faster-by-copying-your-projects-views-custom-fields-and-draft-issues)!

You currently cannot copy issues or pull requests, only draft issues as an option. Looking through this discussion, is there anything else you would like to include when copying a project? Given issues and workflows are noted 😁
